src/test/java/itm/oss/splitter/ExporterTest.java

```diff
@@ -0,0 +1,55 @@
+package itm.oss.splitter;
+
+import org.junit.jupiter.api.Test;
+import java.io.File;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.*;
+import java.math.*;
+
+public class ExporterTest {
+
+    @Test
+    public void testWritePaymentsCsv_createsFileWithHeader() throws Exception {
```

Comment:
I recommend replacing this method name to writePaymentsCsvCreatesFileWithHeader.

1. 'Test' can be deleted. Annotation '@Test' implies that this method is for the test.
2.  Camel case is standard rather than underscore in Java.
```suggestion
    public void writePaymentsCsvCreatesFileWithHeader throws Exception {
```

Reply:
Thanks for your feedback!
I’ve renamed the test method to removed the redundant ‘test’.
The updated change has been pushed to the branch.

Reply:
LGTM! Thanks for the fast reply. I recognized that you have changed the function name that I missed. Now your code is easier to read.